StellarApp/Controllers/NewsViewController.swift

```diff
@@ -1,30 +1,72 @@
-//
-//  NewsViewController.swift
-//  StellarApp
-//
-//  Created by TingxinLi on 4/8/19.
-//  Copyright © 2019 TingxinLi. All rights reserved.
-//
-
 import UIKit
 
 class NewsViewController: UIViewController {
 
+    private var articles = [ArticleWrapper]() {
+        didSet {
+            DispatchQueue.main.async {
+                self.newsView.NewsCollectionView.reloadData()
+            }
+        }
+    }
+    
+    
+    let newsView = NewsView()
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        view.addSubview(newsView)
+        let firstColor = UIColor.init(red: 255/255, green: 0/255, blue: 204/255, alpha: 1)
+        let secondColor = UIColor.init(red: 51/255, green: 51/255, blue: 153/255, alpha: 1)
+        let gradient = CAGradientLayer()
+        gradient.frame = self.newsView.NewsCollectionView.bounds
+        gradient.colors = [firstColor.cgColor, secondColor.cgColor]
+        self.newsView.layer.insertSublayer(gradient, at: 0)
+        newsView.NewsCollectionView.delegate = self
+        newsView.NewsCollectionView.dataSource = self
+        getArticles(keyword: "planets")
         // Do any additional setup after loading the view.
     }
     
+    private func getArticles(keyword: String) {
+        ApiClient.getNews(query: keyword) { (error, data) in
+            if let error = error {
+                print(error.errorMessage())
+            } else if let data = data {
+                self.articles = data
+            }
+        }
+    }
+    
+}
 
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+extension NewsViewController: UICollectionViewDataSource, UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return articles.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "NewsCell", for: indexPath) as? NewsCollectionViewCell else { return UICollectionViewCell() }
+        let thisArticle = articles[indexPath.row]
+        cell.articleLabel.text = "Title: \(thisArticle.title)"
```

Comment:
See [this comment](https://github.com/vaslee/Stellar/pull/10#discussion_r276283168)